paddle/fluid/operators/optimizers/momentum_op.h

```diff
@@ -123,9 +138,13 @@ class CPUDenseMomentumFunctor {
 
     auto p = framework::EigenVector<T>::Flatten(*param);
     auto v = framework::EigenVector<T>::Flatten(*velocity);
-    auto g = framework::EigenVector<T>::Flatten(*grad);
+    auto g = framework::EigenVector<T>::Flatten(*const_cast<Tensor*>(grad));
```

Comment:
Don't recommend to use `const_cast`. Can you avoid this kind of operation?

Reply:
thanks.

Reply:
Already fixed.